amplify.yml

```diff
@@ -48,8 +61,13 @@ frontend:
         - key: "Referrer-Policy"
           value: "same-origin"
         - key: "Content-Security-Policy"
-          value:
-            "connect-src 'self' https://*.postman.gov.sg https://postmangovsg-dev-upload.s3.ap-northeast-1.amazonaws.com/ https://postmangovsg-prod-upload.s3.ap-northeast-1.amazonaws.com https://www.google-analytics.com;
+          value: "connect-src 'self' \
+            https://*.postman.gov.sg https://postmangovsg-dev-upload.s3.ap-northeast-1.amazonaws.com/ \
+            https://postmangovsg-prod-upload.s3.ap-northeast-1.amazonaws.com \
+            https://o399364.ingest.sentry.io/;
+            report-uri \
+            https://o399364.ingest.sentry.io/api/5261015/security/?sentry_key=06ac5522b7354608a16912921d15ef4e \
```

Comment:
why are there two report uri

Reply:
There are 2 security header endpoints for staging and production each. This allows us to capture any CSP violations (https://docs.sentry.io/error-reporting/security-policy-reporting/#content-security-policy)

Reply:
Yea but now a csp violation on staging gets reported on production as well.  
Btw why do we have 1 project for prod and 1 project for staging instead of 1 project and 2 environments? https://docs.sentry.io/enriching-error-data/environments/

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/report-uri
Maybe we should just remove it. It's very deprecated. 

I attempted to set the uri using the sentry_dsn env var in amplify.yml, but it didn't resolve. report-uri also isn't allowed in the meta tags. Can't see any way around this